### PR TITLE
Application/chatbot with tts

### PR DESCRIPTION
**Pull request by akashAD98**
this project usase 
Hybrid Search 
Full-Text Search (FTS) 
Colbert Reranker 
Langchain Prompts -
Parler Text-to-Speech (TTS) -

user can disable enable audio option also

### CODE REVIEW

**Comment by akashAD98** [comment]
image

**Review by PrashantDixit0 — COMMENTED** [review]
Link of folder is not correct.
Please put image in assets folder

**Review by PrashantDixit0 — COMMENTED** [review]
Folder link and please use image from assets folder

File: applications/Chatbot_with_Parler_TTS/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Chat with PDF using LanceDB and Paler TTS
+This application integrates a PDF chat functionality using LanceDB with advanced RAG (Retrieval-Augmented Generation) methods and
+leverages the Paler Text-to-Speech (TTS) model for audio output.
+
+It is designed to enable high-quality text and speech interaction with PDF documents.
+![image](https://github.com/akashAD98/vectordb-recipes/assets/62583018/7b4edbbe-cde7-44cb-9a70-c79c91df68a1)
```
Comment (PrashantDixit0) [inline applications/Chatbot_with_Parler_TTS/README.md:6]: use relative path for image

File: README.md [hunk 2]
```diff
@@ -87,6 +87,11 @@ These are ready to use applications built using LanceDB serverless vector databa
 | [ GTE MLX RAG ](https://github.com/lancedb/vectordb-recipes/tree/main/applications/GTE_mlx_RAG) | mlx based RAG model using lancedb api support | ![image](./assets/rag-mlx.png)|
 | [Multilingual-RAG](https://github.com/lancedb/vectordb-recipes/tree/main/applications/Multilingual_RAG/) | Multilingual RAG with cohere embedding & support 100+ languages|![image](https://github.com/akashAD98/vectordb-recipes/assets/62583018/be65eb39-25c4-4441-98fc-6ded09689819)|
 | [Multimodal Myntra Fashion Search Engine](https://github.com/ishandutta0098/lancedb-multimodal-myntra-fashion-search-engine) | This app uses OpenAI's CLIP to make a search engine that can understand and deal with both written words and pictures.|![image](./assets/myntra-search-engine.png)|
+| [Advance Chatbot with Parler TTS ](https://github.com/lancedb/vectordb-recipes/tree/main/applications/chatbot_with_TTS/applications/Chatbot_with_Parler_TTS) | This Chatbot app uses Lancedb Hybrid search, FTS & reranker method with Parlers TTS library.|![image](https://github.com/akashAD98/vectordb-recipes/assets/62583018/f076e6b3-e78f-45cf-a461-08d577c31720)|
+
+
+
+chatbot_with_TTS/applications/Chatbot_with_Parler_TTS
```
Comment (PrashantDixit0) [inline README.md:94]: remove this line

File: applications/Chatbot_with_Parler_TTS/README.md (hunk 1 above)
Comment (akashAD98) [reply applications/Chatbot_with_Parler_TTS/README.md:6]: ok got it

**Comment by AyushExel** [comment]
@PrashantDixit0 can I merge this now?

**Comment by PrashantDixit0** [comment]
Links are still not updated, this will result in breaking links in the future

**Comment by akashAD98** [comment]
@AyushExel @PrashantDixit0 its fixed now

**Comment by AyushExel** [comment]
ok merging this

**Pull request merged**